tests/Integration/tests/configuration/input/guides.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<guides xmlns="https://www.phpdoc.org/guides"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="https://www.phpdoc.org/guides vendor/phpdocumentor/guides-cli/resources/schema/guides.xsd"
+        links-are-relative="true"
```

Comment:
According to the XSD that attribute doesn't exist?

Also, these are listed but not set in the example:

* input
* input-file
* output
* input-format
* log-path
* fail-on-log
* show-progress
* theme

Do we want to expose these?

Reply:
I must check that, there might be a problem in the xsd file then.  links-are-relative should exist.

In our usecase it does not make much sense to set the other values. We are either setting them in the command (--output-path=xy) or in the global guides.xml or the default is good to go.

Reply:
The open issues should be resolved by https://github.com/phpDocumentor/guides/pull/666